src/api/flaskr/service/learn/runscript_v2.py

```
@@ -138,85 +151,186 @@ def run_script(
+            reload_generated_block_bid=reload_generated_block_bid,
+            preview_mode=preview_mode,
+            stop_event=stop_event,
+        )
+
+        def producer():
+            try:
+                for item in res:
+                    if stop_event.is_set():
+                        break
+                    output_queue.put(("data", item))
+            except Exception as exc:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)